README.md

```diff
@@ -24,236 +25,21 @@ We do not collect any personal information. If you wish to disable telemetry
 or read more about our telemetry practices please see
 our [telemetry](crates/pop-telemetry/README.md) documentation.
 
-## Getting Started
+## Pop CLI commands
```

Comment:
We have a section below `Learn how to develop with Pop CLI `that already has links to the docs. It seems redundant to have another section with links to the same docs site. 
Can we  refactor this into a single line instead of a whole section? An example I like is from Hardhat:  
 https://github.com/NomicFoundation/hardhat?tab=readme-ov-file#documentation